coalib/output/ConsoleInteraction.py

```diff
@@ -664,22 +667,22 @@ def choose_action(console_printer, actions, apply_single=False):
                 actions_desc_len = len(actions_desc)
                 if c.isnumeric():
                     for i, action in enumerate(actions, 0):
-                        c = int(c)
-                        if i == c:
+                        if i == int(c):
                             actions_desc.append(action.desc)
                             actions_name.append(action.name)
                             break
                 elif c.isalpha():
                     c = c.upper()
-                    c = '(' + c + ')'
+                    str_c = '(' + c + ')'
```

Comment:
don't change this

Reply:
The reason for this change was to preserve the choice entered by the user so that it may be printed later if it turns out to have been an invalid choice. Should I still change this? I will need to store the original choice either way